build.sbt

```diff
@@ -444,18 +444,18 @@ lazy val scalaTck = project
 lazy val codegenCore =
   project
     .in(file("codegen/core"))
-    .enablePlugins(AkkaGrpcPlugin, PublishSonatype)
+    .enablePlugins(PublishSonatype, sbtprotoc.ProtocPlugin)
     .settings(common)
     .settings(
       name := "kalix-codegen-core",
       testFrameworks += new TestFramework("munit.Framework"),
       Test / fork := false)
     .settings(Dependencies.codegenCore)
-    .settings(Compile / akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Java))
     .settings(
       Compile / javacOptions ++= Seq("--release", "11"),
       Compile / scalacOptions ++= Seq("-release", "11"),
-      scalaVersion := Dependencies.ScalaVersionForTooling)
+      scalaVersion := Dependencies.ScalaVersionForTooling,
+      Compile / PB.targets := Seq(PB.gens.java -> (Compile / sourceManaged).value))
```

Comment:
Shouldn't this be for tests only rather than main code? Is it something we distribute (kalix protocol classes?)

Reply:
The codegen classes have references to generated Kalix annotations. 
https://github.com/lightbend/kalix-jvm-sdk/blob/18d2ef48e5188bb52f2ccc3a7b33a4bebe9f3988/codegen/core/src/main/scala/kalix/codegen/ModelBuilder.scala#L26-L32